google-cloud-logging/lib/google/cloud/logging/async_writer.rb

```diff
@@ -237,18 +240,21 @@ def wait! timeout = nil
         ##
         # Stop this asynchronous writer and block until it has been stopped.
         #
-        # @param [Number] timeout Timeout in seconds.
+        # @param [Number, nil] timeout The maximum number of seconds to wait for
+        #   shutdown to complete. The default value is `nil`.
```

Comment:
Document what `nil` means (I assume infinite timeout?)

Reply:
I wish it were documented in [Concurrent::FThreadPoolExecutor#await_for_termination](https://www.rubydoc.info/gems/concurrent-ruby/1.0.2/Concurrent%2FThreadPoolExecutor%3Await_for_termination), but it's not. My observation is that it is the same as a `0` timeout, but I'm not 100% certain. :(

Reply:
@quartzmo It ends up here: https://github.com/ruby-concurrency/concurrent-ruby/blob/master/lib/concurrent-ruby/concurrent/atomic/event.rb#L78-L92 which says it will wait forever if no (or `nil`) timeout is given.

Reply:
Thanks for the link. I'll look into it.

Reply:
Updated, PTAL!